server/libs/infra/src/db/migrations/1677613712565-AlbumThumbnailRelation.ts

```diff
@@ -0,0 +1,60 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class AlbumThumbnailRelation1677613712565 implements MigrationInterface {
+  name = 'AlbumThumbnailRelation1677613712565';
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    // Make sure all albums have a valid albumThumbnailAssetId UUID or NULL.
+    await queryRunner.query(`
```

Comment:
This is the manual query for migration, not from the generated command, correct?

Reply:
This is the query that gets generated by `AlbumRepository.updateThumbnails()` with one added `::varchar` cast at the end. The query below this one is also custom, because the generated command removes and then re-adds the column which would result in data loss.